SConstruct

```diff
@@ -196,20 +196,28 @@ Export('command_line', 'env', 'whereis', 'revaction')
 SConscript('doc/SConscript')
 
 
-# Copy manpage's into base dir for inclusion in pypi packages
+# Copy manpages into base dir for inclusion in pypi packages
 man_pages = env.Install("#/", Glob("#/build/doc/man/*.1"))
 
-# Build packages for pypi
-wheel = env.Command('$DISTDIR/SCons-${VERSION}-py3-none-any.whl', ['setup.cfg', 'setup.py', 'SCons/__init__.py']+man_pages,
-            '$PYTHON setup.py bdist_wheel')
+# Build packages for pypi. 'build' makes sdist (tgz) + whl
+wheel = env.Command(
+    target=[
+        '$DISTDIR/SCons-${VERSION}-py3-none-any.whl',
+        '$DISTDIR/SCons-${VERSION}.tar.gz',
+    ],
+    source=['setup.cfg', 'setup.py', 'SCons/__init__.py'] + man_pages,
+    action='$PYTHON -m build --outdir $DISTDIR',
+)
 
-zip_file = env.Command('$DISTDIR/SCons-${VERSION}.zip', ['setup.cfg', 'setup.py', 'SCons/__init__.py']+man_pages,
-            '$PYTHON setup.py sdist --format=zip')
-tgz_file = env.Command('$DISTDIR/SCons-${VERSION}.tar.gz', ['setup.cfg', 'setup.py', 'SCons/__init__.py']+man_pages,
-            '$PYTHON setup.py sdist --format=gztar')
+# Do we actually need this one?
```

Comment:
Historically we've always included a zip file of the tree for use by windows.
So I'd say yes.

Reply:
I restored that bit, but just FWIW, a Windows user can `pip install` directly from the `.tar.gz` sdist so it's not clear *that* zipfile provides anything extra.

Reply:
If they unzip it and edit it..  or if they still want to use setup.py..